packages/react-components/src/localization/locales/de-DE/strings.json

```diff
@@ -169,6 +172,7 @@
     "audioDeviceMenuTooltip": "Audiogerät auswählen",
     "microphoneMenuTitle": "Mikrofon",
     "microphoneMenuTooltip": "Mikrofon auswählen",
+    "defaultMicrophoneLabelFallback": "Standard",

```

Comment:
Is this the right translation here?

Reply:
Apparently 🤷

![image](https://github.com/user-attachments/assets/85e2562d-24bc-408a-a2e5-e59edf69e7a7)

cc @DominikMe 😄